src/core/client/stream/tabs/Comments/PermalinkView/ConversationThreadContainer.tsx

```diff
@@ -213,6 +213,7 @@ const ConversationThreadContainer: FunctionComponent<Props> = ({
               <RejectedTombstoneContainer comment={comment}>
                 <DeletedTombstoneContainer comment={comment}>
                   <CommentContainer
+                    parentLoaded={false}
```

Comment:
this shouldn't be `false`, but in this case with the `parentLoaded` property: `remaining === 0`